lib/rust/ensogl/core/src/display/render/pass.rs

```diff
@@ -170,7 +173,7 @@ impl OutputDefinition {
     /// Constructor of the RGBA u8 output with default texture parameters. It is the most popular
     /// option and you should use it to render colors with your passes.
     pub fn new_rgba<Name: Str>(name: Name) -> Self {
-        let internal_format = texture::Rgba;
+        let internal_format = texture::Rgba8;
```

Comment:
Wait, if we've been using wrong format, how on earth was it working? :O

Reply:
`Rgba` isn't wrong; used with `u8` channel values, it is usually equivalent to using `Rgba8`. However, in some situations the more specific `Rgba8` value can produce a complete texture, when `Rgba` would not; and there are driver (and browser)-specific differences between handling of incomplete textures... So, although there should be more difference here, I think it is safest to provide the most specific value possible.